qemu/tests/cfg/numa.cfg

```diff
@@ -11,22 +15,21 @@
             only Linux
             type = numa_consistency
             start_vm = no
-            threshold = 0.05
-            ppc64,ppc64le:
-                threshold = 0.15
+            mem_fixed = 4096
+            backend_mem = "memory-backend-ram"
+            size_mem = ${mem_fixed}M
```

Comment:
For machine mem object, it is useless because:
https://github.com/avocado-framework/avocado-vt/blob/c1933f3149abda3f7e73140831c6aa97483ac632/virttest/qemu_vm.py#L1099C17-L1099C17

Reply:
Updated, thanks !